packages/frontend/package.json

```diff
@@ -44,6 +44,7 @@
         "@sentry/browser": "^6.4.1",
         "big.js": "^6.2.1",
         "bip39-light": "^1.0.7",
+        "calimero-wallet-utils": "file:/Users/ijerkovic/Dev/calimero/calimero-wallet-utils/lib",
```

Comment:
@xilosada @miraclx Will update this to the published package, just added the PR here to verify all is good